dashboard/src/main/resources/webapp/static/lib/angular-topology-editor/canvas.js

```diff
@@ -101,7 +101,8 @@ Graph.Node.popovercontent = function(i) {
 };
 
 var Canvas = (function() {
-
+    var LOG = Log.getLogger("Canvas").setLevel(Log.INFO);
```

Comment:
log.js is the library I wrote, but it has a big disadvantage: you don't know where was the log statement. That's why I switched to loglevel. I should have removed log.js.

Regarding "log" being a global variable, we can overcome it with:
https://github.com/pimterry/loglevel#with-noconflict